validation/src/main/scala/hmda/validation/engine/lar/quality/LarQualityEngine.scala

```diff
@@ -7,6 +7,11 @@ import hmda.validation.rules.lar.quality._
 
 trait LarQualityEngine extends LarCommonEngine with ValidationApi {
 
+  private def q022(lar: LoanApplicationRegister): LarValidation = {
+    // TODO: 2017 is hardcoded for now.  It will be passed in through the API at a later date
```

Comment:
So, what is the definition of "done" that we're using here? At a first glance, I don't see how we can call it done if it's not hooked up. Is there something blocking us on completing the feature now?